### PR TITLE
Eggdice/skid add documentation

### PR DESCRIPTION
**Pull request by EggDice**
Minimal documentation and version number is added to able to show Sony that we have something official :)

Related issues: #12 #13

### CODE REVIEW

File: rfc/006-skid.mdx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+- External Services: Reusable services that primarily serve as adapters for libraries and
+  platform-related APIs.
+- Dev Tools: A collection of tools to facilitate development, such as feature flippers and test
+  environment management.
+
+## Documentation
+
+Documentation for skid will be hosted under the following domain: `skid.bobcatscoding.com`. It will
+include comprehensive details on how to use the library, its components, and any dependencies.
+Distribution
```
Comment (Fruzsi555) [inline rfc/006-skid.mdx:27]: it shall be a new header

File: rfc/006-skid.mdx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+- Utils: A set of utility functions and object creators to assist with common development tasks.
+- External Services: Reusable services that primarily serve as adapters for libraries and
+  platform-related APIs.
+- Dev Tools: A collection of tools to facilitate development, such as feature flippers and test
+  environment management.
+
+## Documentation
+
+Documentation for skid will be hosted under the following domain: `skid.bobcatscoding.com`. It will
+include comprehensive details on how to use the library, its components, and any dependencies.
```
Comment (Fruzsi555) [inline rfc/006-skid.mdx:26]: delete comma before 'and'

File: rfc/006-skid.mdx [hunk 3]
```diff
@@ -0,0 +1,63 @@
+# `skid`
+
+This document proposes the creation and maintenance of a shared library called `skid` for use across
+all Bobcats Coding products. The goal is to standardize and simplify the development process,
+promote code reuse, and improve maintainability by centralizing common code, tools, and services.
```
Comment (Fruzsi555) [inline rfc/006-skid.mdx:5]: delete comma before 'and'

File: rfc/006-skid.mdx [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+The `skid` library is openly available as a GitHub repository, mirrored from our monorepo's `skid/`
+directory using "git subtree". This will enable us to use the library in projects that cannot be
+placed in our monorepo. Maybe someday even it enables external contributors to access the code and
+participate in the development process.
+
+## Command for Pushing and Publishing
+
+An NPM command should be provided to push and publish the library, simplifying the process of
+updating the library on the npm registry.
```
Comment (Fruzsi555) [inline rfc/006-skid.mdx:47]: NPM registry

File: rfc/006-skid.mdx [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+## License
+
+The library is released under the MIT License, allowing for free and open-source use, modification,
+and distribution.
+
+## Repository
+
+The `skid` library is openly available as a GitHub repository, mirrored from our monorepo's `skid/`
+directory using "git subtree". This will enable us to use the library in projects that cannot be
+placed in our monorepo. Maybe someday even it enables external contributors to access the code and
```
Comment (Fruzsi555) [inline rfc/006-skid.mdx:41]: instead of 'Maybe someday' could be written: 'It is planned for the future'

File: rfc/006-skid.mdx (hunk 1 above)
Comment (EggDice) [reply rfc/006-skid.mdx:27]: Ohh, thank you! As you can see my attention to detail is not the best <3

File: rfc/006-skid.mdx (hunk 5 above)
Comment (EggDice) [reply rfc/006-skid.mdx:41]: I would not say it is planned, we just wish that happens, and it would be amazing.

**Review by Fruzsi555 — APPROVED** [review]

**Pull request merged**